### ISSUE TITLE
{ [Error: Command failed: ] killed: false, code: 1, signal: null }

### ISSUE THREAD

**Issue by testvgs7** [leftshifters/excel-parser#20]
Hi am getting an error while importing .xlsx files, pls anybody help out resolve this issue,,

{ [Error: Command failed: ] killed: false, code: 1, signal: null }

var excelParser = require('excel-parser');
        excelParser.worksheets({
          inFile: 'file'
        }, function(err, worksheets){
          if(err) console.error(err);
          console.log(worksheets);
        });

**Comment by nassan** [comment]
Is it possible you have a Python 3.x.x directory in your path as well? I had installed Python 2.7.x, but because Python 3.x.x was earlier in my path variable, excel-parser was actually calling Python 3.x.x, and giving me this error.

**Comment by sthawali** [comment]
@nassan You need to change the path variable. follow this,
https://docs.python.org/2/using/windows.html#excursus-setting-environment-variables

**Comment by nassan** [comment]
Right, I did remove the python 3.x.x from the path variable, and it worked fine. :)  I thought, perhaps someone else might have stumbled in the same way.

Thank you for your prompt reply.
